builder/docker/config.go

```diff
@@ -39,10 +39,17 @@ type Config struct {
 	// for work [file provisioner](/docs/provisioners/file). This defaults
 	// to c:/packer-files on windows and /packer-files on other systems.
 	ContainerDir string `mapstructure:"container_dir" required:"false"`
+	// An array of devices which will be accessible in container when it's run
```

Comment:
I think it would be good to link directly to the docker docs on the command line flags these correspond to, so that users know what the underlying mechanism being used is.

Reply:
Good catch, I have added link to the docker documentation